.gitignore

```diff
@@ -0,0 +1,2 @@
+## PHPStorm
```

Comment:
The correct spelling is "[PhpStorm](https://www.jetbrains.com/phpstorm/#:~:text=Download-,PhpStorm,-Empowering%20PHP%20Developers)". 🙈 🤓
```suggestion
## PhpStorm
```